exporter/splunkhecexporter/client.go

```diff
@@ -18,14 +18,14 @@ import (
 	"bytes"
 	"compress/gzip"
 	"context"
-	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"sync"
 
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Please provide the motivation for this extra dependency

Reply:
See https://github.com/json-iterator/go for a bit of background.

Jsoniter is a faster JSON processor.

In this change, I also remove newline characters between elements to reduce network traffic ever so slightly as well.